OpenSim/Simulation/GeodesicWrap/ImplicitSphere.h

```diff
@@ -0,0 +1,33 @@
+#ifndef IMPLICIT_SPHERE_H_
+#define IMPLICIT_SPHERE_H_
```

Comment:
To support multiple surfaces, perhaps this file becomes `ImplicitSurface` and contains the `ImplicitSurfaceParamsImpl` for all surfaces.

Reply:
Yeah, I apparantly I went a bit overboard with the pointers in boxes...